tests/system/Log/LoggerTest.php

```diff
@@ -427,16 +426,6 @@ public function testLogLevels(): void
         $this->assertSame($expected, $logs[0]);
     }
 
-    public function testNonStringMessage(): void
-    {
-        $this->expectException(TypeError::class);
-
-        $config = new LoggerConfig();
-        $logger = new Logger($config);
-
-        $logger->log(5, $config);
-    }
-
```

Comment:
This test is no longer needed as $message is now typed natively as `string|Stringable` so this would automatically throw a TypeError.